src/todomanager/popup.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react';
+
+function Popup() {
+  const Alert = alert;
+  // 임시로 useState로 데이터 저장 [indexedDB로 데이터를 저장하고 불러올 수 있게 작업 예정]
+  const [todos, setTodos] = useState([
+    {
+      id: 0,
+      content: '할 일 1',
+      dueDate: new Date().toISOString().split('T')[0],
+      done: false,
+    },
+  ]);
+  const [newTodo, setNewTodo] = useState('');
+
+  // 임시로 id값은 배열의 길이로 설정
+  const addTodo = () => {
+    if (typeof newTodo === 'string' && newTodo.length > 0) {
+      setTodos([
+        ...todos,
+        {
+          id: todos.length,
+          content: newTodo,
+          dueDate: new Date().toISOString().split('T')[0],
+          done: false,
+        },
+      ]);
+      setNewTodo('');
+    } else {
+      Alert('할 일을 입력하세요.');
+    }
+  };
+
+  const completeTodo = (id: number) => {
+    setTodos(
+      todos.map(todo => {
+        if (todo.id === id) {
+          return { ...todo, done: !todo.done };
+        }
+        return todo;
+      }),
+    );
+    todos.forEach(todo => {
+      if (todo.id === id && !todo.done) {
+        Alert(`${todo.content} 완료!`);
+      }
+    });
+  };
+
+  return (
+    <div>
+      <div className="w-32 h-10">
+        <h1>투두막</h1>
+      </div>
+      <div className="flex-col w-32 h-32">
+        <div id="todo-list" className=" h-16 overflow-y-scroll">
+          {todos.map(todo => (
+            <div key={todo.id} className="flex">
+              <div>{todo.content}</div>
+              <input
+                type="checkbox"
+                checked={todo.done}
+                onChange={() => completeTodo(todo.id)}
```

Comment:
Suggest: value 값으로 추적이 가능하지 않을까요?

Reply:
```suggestion
  value={todo.id}
  onChange={e => completeTodo(Number(e.target.value))}
````

이렇게 처리하는 것을 말씀하시는거 맞나요?

Reply:
아니요. 

```tsx
const completeTodo = (e:알아서 타입 찾으세요) => {
   setSome(e.target.value)
}
```

Reply:
수정했습니다!

해당 작업: [19d297c](https://github.com/ToyMakers/todo-mark/pull/9/commits/19d297c98233660c2f9000eab1b1b8b9e0d7a6f3)